hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/CloudInfoCollector.java

```diff
@@ -91,6 +91,12 @@ public void forEachMetric(Node node, BiConsumer<PhoneHomeMetrics, String> metric
         } catch (IOException e) {
             info.put(PhoneHomeMetrics.DOCKER, "N");
         }
+
+        String cloudEnv = System.getenv("HZ_CLOUD_ENVIRONMENT");
```

Comment:
let's consider `null` value of `cloudEnv` as `false` so we have only two values in the column, it will simplify the analysis, WDYT?

Reply:
@promanenko on the contrary - i made it a "nullable string" intentionally so that you can query cloud info with only one field:

- vrd == null (not a cloud member)
- vrd != null (a cloud member; and you can query further via comparing values in the description)

otherwise we'll need two fields like .isThisCloud and .whatKindOfCloud.

Reply:
... i think we meant the same thing :)